casepro/settings_common.py

```diff
@@ -303,10 +303,14 @@
 
     'orgs.org': ('create', 'update', 'list', 'edit', 'home', 'inbox'),
 
-    'msgs.label': ('create', 'update', 'list', 'unlabelled'),
+    'msgs.label': ('create', 'update', 'list'),
+
+    'msgs.message': ('action', 'bulk_reply', 'forward', 'label', 'history', 'search', 'unlabelled'),
```

Comment:
Previously message wasn't a local model so we couldn't have these permissions so all message related views just used `orgs.org_inbox`. Now we can get a bit more specific.